.github/workflows/ubuntu-20.04.yml

```diff
@@ -1,20 +1,23 @@
 # MLOS continuous integration pipelines:
+# vim: set ft=yaml:
 #
-# - Build/Test on Windows
+# - Build/Test on Windows (split to windows.yml)
 # - Build/Test Ubuntu 16.04, 18.04, 20.04
 #   (using docker images which also get published on success)
 # - Check for license headers
 # - Generate a static website (and link check it)
+#
+# Note: to allow separate retries, we split these into different workflows using the gen-workflows.sh script.
 
-name: CI
+name: CI-Ubuntu-20.04
```

Comment:
This is a templated out version of this file for Ubuntu 20.04.  Git seems to match against it instead of `ubuntu.yml.tmpl` which is the source it's derived from.  🤷